modules/libcom/src/yacc/output.c

```diff
@@ -600,10 +601,16 @@ pack_vector(int vector)
 
                 newmax = maxtable;
                 do { newmax += 200; } while (newmax <= loc);
-                table = (short *) REALLOC(table, newmax*sizeof(short));
-                if (table == 0) no_space();
```

Comment:
Are there enough instances of this pattern to justify a helper function?  Maybe a `static inline` in `defs.h` where `REALLOC()` is defined?

Reply:
These `REALLOC()` calls actually look to be fine even in the face of OOM errors. The `no_space()` routine is marked `NORETURN` and calls `exit()` via `done()`. Is there no way to teach these analysis tools about `__attribute__((noreturn))`, maybe by adjusting the `#ifdef __GNUC__` around the `NORETURN` definitions?

Reply:
Good point.  So the `libCom/yacc: handle realloc() failures correctly` commit is based on a false positive.